app-services/secrets/SecretsExample.postman_collection.json

```diff
@@ -104,15 +63,15 @@
 				],
 				"body": {
 					"mode": "raw",
-					"raw": "{\n  \"path\" : \"/mongodb\",\n  \"secrets\" : [\n    {\n      \"key\" : \"username\",\n      \"value\" : \"mongo-user\"\n    },\n    {\n      \"key\" : \"password\",\n      \"value\" : \"SuperDuperSecretPassword\"\n    }\n  ]\n}",
+					"raw": "{\n    \"id\": \"5bd794de0e36080001f53eab\",\n    \"pushed\": 0,\n    \"device\": \"Random-Float-Device\",\n    \"created\": 1540855006481,\n    \"modified\": 0,\n    \"origin\": 1540855006469,\n    \"schedule\": null,\n    \"event\": null,\n    \"readings\": [\n        {\n            \"id\": \"5bd794de0e36080001f53eac\",\n            \"pushed\": 0,\n            \"created\": 1540855006481,\n            \"origin\": 1540855006469,\n            \"modified\": 0,\n            \"device\": \"Random-Float-Device\",\n            \"name\": \"HoldingRegister_8454\",\n            \"value\": \"75.30\"\n        }\n    ]\n}",
 					"options": {
 						"raw": {
 							"language": "json"
 						}
 					}
 				},
 				"url": {
-					"raw": "http://localhost:48095/api/v1/secrets",
+					"raw": "http://localhost:48095/api/v1/trigger",
```

Comment:
This is using the HTTP trigger, not message bus so the description previously about Core Data is wrong.

Reply:
good catch